### ISSUE TITLE
QUESTION: Is this project live?

### ISSUE THREAD

**Issue by thejessicafelts** [Mitzi-S-Soto/SolarSystem#2]
Is this project live? Do you have a GitHub page where this project lives or will live (as you do with the slider, for example)? I would love to see this project in action. Also, apparently GitHub makes you submit issues to ask questions (even though from the back-end you'll be able to label it as a question (though I can't, from here), which is odd.

**Comment by Mitzi-S-Soto** [comment]
Nothing live yet as it does nothing yet and is not finished. When it gets closer to completion it will. Github seems like a reasonable place to host projects like this rather than on my own site.

**Comment by thejessicafelts** [comment]
Okay, cool. I would suggest setting up your GitHub Pages site (you're on Mitzi-S-Soto.github.io I think?), with a page specifically for projects, and then link out from there. It keeps everything in once place, and is easy to find for people who come to your profile and want to learn more about and see what you've done. 😄

**Comment by Mitzi-S-Soto** [comment]
Yeah, I wanted to muck with that actually but wasn't sure how to set up the base mitzi-s-soto.github.io site....I will be looking into it tho
